Dev

LunexCoding opened the pull request.

LunexCoding committed:
Init repo

LunexCoding committed:
Upload files

oleksandr-grabovyi commented on src/updater/updater.py:
```diff
@@ -0,0 +1,132 @@
+
+    def run(self):
+        _log.debug("Updater запущен...")
+        self._ftp = g_ftpClient
```
зачем хранить ссылку на глобальную перменную?

oleksandr-grabovyi commented on src/updater/preinit.py:
```diff
@@ -0,0 +1,25 @@
+from common.logger import logger
+
+
+def createLog():
```
preinit это файл который должен вызваться 1 раз при инициализации всего приложения. Скажем так. Выделили что-то, что грузится первее остального. Для организации порядка загрузки. Тут не может быть функций и чего-то такого

oleksandr-grabovyi commented on src/updater/consts.py:
```diff
@@ -0,0 +1,16 @@
+import sys
+
+
+class Constants:
```
а зачем класс константы файлу константы? ну это просто странно

oleksandr-grabovyi commented on src/shared/tools/ftp.py:
```diff
@@ -0,0 +1,209 @@
+)
+
+
+if __name__ == "__main__":
```
это самостоятельный файл? откуда тут меин возьмется?

oleksandr-grabovyi commented on src/server/preinit.py:
```diff
@@ -0,0 +1,20 @@
+from config import g_settingsConfig
+from common.logger import logger
+
+def createLog():
```
да по идеи без функции. Просто тут же сразу createLog

oleksandr-grabovyi commented on src/server/main.py:
```diff
@@ -0,0 +1,50 @@
+
+
+_log = logger.getLogger(__name__)
+LAUNCH_COMMANDS = [-1, -2]
```
что -1, -2?

oleksandr-grabovyi commented on src/server/dataStructures/referenceBook.py:
```diff
@@ -0,0 +1,156 @@
+        self._columns = self._getTableColumns()
+        self._columnsForInsertion = ColumnsForInsertion.getColumns(self._table)
+
+    def _getTableColumns(self):
```
как-то у тебя в перемешку совсем. Обычно сначала конструктора и инициализация, потом публичные методы, защищенные, приватные. А тут чет вразнобой

oleksandr-grabovyi commented on src/client/ui/contexts/consts.py:
```diff
@@ -0,0 +1,17 @@
+from network.tables import DatabaseTables
+
+
+class Constants:
```
почему оно все в 1 классе который называется так же как и файл?))

LunexCoding committed:
Refactoring imports and constants

LunexCoding committed:
Logic Inversion and Refactoring

LunexCoding committed:
Обновлены настройки подмодуля

LunexCoding committed:
Удален подмодуль src/Logger

LunexCoding committed:
Add socket notifications

Added webparser
Improved logging
Added local commands to the client - called via socket notifications
Module ftp moved from shared to common
Separated common config, located in common
Added preinit for initializing loggers

LunexCoding committed:
Fix logging
